pkg/core/oracle_test.go

```diff
@@ -31,6 +39,98 @@ import (
 
 var oracleModulePath = filepath.Join("..", "services", "oracle")
 
+// getTestContractState returns test contract which uses oracles.
+func getOracleContractState(h util.Uint160, stdHash util.Uint160) *state.Contract {
```

Comment:
Can we compile it and make common between modules? Duplication doesn't look good.

Reply:
Yes, fixed.